airflow/providers/google/common/hooks/base_google.py

```diff
@@ -674,7 +674,10 @@ async def acquire_access_token(self, timeout: int = 10) -> None:
 
         self.access_token = cast(str, self.credentials.token)
         self.access_token_duration = 3600
-        self.access_token_acquired_at = datetime.datetime.now(tz=datetime.timezone.utc)
+        # access_token_acquired_at is specific to gcloud-aio's Token. On subsequent calls of `get` it will be used
+        # with `datetime.datetime.utcnow()`. Therefore we have to use an offset-naive datetime.
+        # https://github.com/talkiq/gcloud-aio/blob/f1132b005ba35d8059229a9ca88b90f31f77456d/auth/gcloud/aio/auth/token.py#L204
+        self.access_token_acquired_at = datetime.datetime.utcnow()  # noqa: TID251
```

Comment:
noqa not an option here, since this related to the deprecated things in Python 

```suggestion
        self.access_token_acquired_at = datetime.datetime.now(tz=datetime.timezone.utc).replace(tzinfo=None)
```

Reply:
yeah, but without upgrading gcloud-aio-auth's major version (which is a big project on its own) we are forced to use the naive version for now, if we do not want to break existing functionality.